debian/copyright

```diff
@@ -1,59 +1,1130 @@
+ separately licensed software that they have included with MySQL.
+ .
+ Without limiting anything contained in the foregoing, this file,
+ which is part of C Driver for MySQL (Connector/C), is also subject to the
+ Universal FOSS Exception, version 1.0, a copy of which can be found at
+ http://oss.oracle.com/licenses/universal-foss-exception.
+
+Files: debian/autobake-deb.sh
+ extra/mysqld_safe_helper.c
+ include/handler_state.h
+ include/json_lib.h
+ include/mysqld_default_groups.h
+ libmysqld/resource.h
+ libservices/my_crypt_service.c
+ mysys/my_setuser.c
+ scripts/convert-debug-for-diff.sh
+ sql-bench/graph-compare-results.sh
+ sql-bench/test-table-elimination.sh
+ sql/message.h
+ sql/opt_trace_context.h
+ sql/proxy_protocol.h
+ sql/rpl_parallel.h
+ sql/win_tzname_data.h
+ strings/ctype-ascii.h
+ support-files/binary-configure.sh
+ support-files/mariadb.logrotate.sh
+ support-files/mini-benchmark.sh
+ support-files/mysql.server-sys5.sh
+ support-files/mysqld_multi.server.sh
+ support-files/wsrep.cnf.sh
+ support-files/wsrep_notify.sh
+License: GPL-2.0
+Comment: These files do not have a GPL 2.0 header.
+ If someone has obligations, please state otherwise.
+
+Files: zlib/adler32.c
+ zlib/compress.c
+ zlib/crc32.c
+ zlib/crc32.h
+ zlib/deflate.c
+ zlib/gzclose.c
+ zlib/gzguts.h
+ zlib/gzlib.c
+ zlib/gzread.c
+ zlib/gzwrite.c
+ zlib/infback.c
+ zlib/inffast.c
+ zlib/inffast.h
+ zlib/inffixed.h
+ zlib/inflate.c
+ zlib/inflate.h
+ zlib/inftrees.c
+ zlib/inftrees.h
+ zlib/trees.c
+ zlib/trees.h
+ zlib/uncompr.c
+ zlib/zlib.h
+ zlib/zutil.c
+ zlib/zutil.h
+Copyright: 1995-2003, 2010, 2014, 2016, Jean-loup Gailly, Mark Adler
+  1995-2003, 2010, Mark Adler
+  1995-2005, 2010, Mark Adler
+  1995-2005, 2014, 2016, Jean-loup Gailly, Mark Adler
+  1995-2011, 2017, Mark Adler
+  1995-2017, Mark Adler
+  1995-2019, Mark Adler
+  1995-2022, Mark Adler
+  1995-2024, Jean-loup Gailly
+  1995-2024, Jean-loup Gailly, Mark Adler
+  1995-2024, Mark Adler
+  2004, 2010, Mark Adler
+  2004-2017, Mark Adler
+  2004-2019, Mark Adler
+  2004-2024, Mark Adler
+  Copyright: 1995-2024, Jean-loup Gailly and Mark Adler
+License: Zlib
+ 
+Files: include/ma_dyncol.h
+ include/queues.h
+ mysys/ma_dyncol.c
+ mysys/queues.c 
+ strings/bchange.c
+ strings/bmove_upp.c
+ strings/int2str.c
+ strings/is_prefix.c
+ strings/llstr.c
+ strings/longlong2str.c
+ strings/my_strtoll10.c
+ strings/str2int.c
+ strings/strappend.c
+ strings/strcend.c
+ strings/strcont.c
+ strings/strend.c
+ strings/strfill.c
+ strings/strmov.c
+ strings/strnlen.c
+ strings/strnmov.c
+ strings/strxmov.c
+ strings/strxnmov.c
+Copyright: 2000, TXT DataKonsult Ab & Monty Program Ab
+  2000, TXT DataKonsult Ab & Monty Program Ab
+  2003, TXT DataKonsult Ab
+  2009-2011, Monty Program Ab
+  2009, 2013, Monty Program Ab
+  2010, Monty Program Ab
+  2011-2012, Oleksandr Byelkin
+  2011, 2017, MariaDB Corporation.
+  2011, Monty Program Ab
+  2011, Oleksandr Byelkin
+  Richard A. O'Keefe
```

Comment:
Would -1984 for Richard be fair based on strings/README? optional.

Reply:
This one is also fixed in new local version but let's see what else should be changes before I'll update.